app/views/layouts/application.html.slim

```diff
@@ -12,7 +12,8 @@ html lang="#{I18n.locale}"
       = content_for?(:title) ? APP_NAME + ' - ' + yield(:title) : APP_NAME
 
     == stylesheet_link_tag 'application', media: 'all'
-    == javascript_include_tag 'application'
+    - unless controller_name == 'home'
+      == javascript_include_tag 'application'
```

Comment:
is this an optimization or a fix/workaround? I'd prefer to see the js just loaded everywhere for simplicity.